trunk/examples/witness-checking/regression/correctness/doubleLoop.c-witness.yaml

```diff
@@ -0,0 +1,54 @@
+-
+  entry_type: loop_invariant
+  metadata:
+    format_version: 0.1
+    uuid: "e92e8d2b-0ac6-4d6d-b6bd-6598f9843c13"
+    creation_time: "2023-08-02T17:29:09+02:00"
+    producer:
+      name: ULTIMATE
+      version: 0.2.3
+    task:
+      input_files:
+        - "C:\ultimate\trunk\examples\witness-checking\regression\correctness\doubleLoop.c"
+      input_file_hashes:
+        "C:\ultimate\trunk\examples\witness-checking\regression\correctness\doubleLoop.c": hash
```

Comment:
has the string `hash` a special meaning for us?

Reply:
I think I created this witness manually, so I have probably overwritten the hash setting. But as already said: metadata are simply ignored for validation.